README.md

```diff
@@ -361,13 +361,34 @@ provided will be used to override the default configuration.
 
 ## Ignoring Rules
 
-If some rule doesn't align with your team's preferences, don't worry! Regal is not meant to be the law, and some rules
-may not make sense for your project, or parts of it. Regal provides several different ways to ignore rules, either
-entirely, or with more granularity.
+If one of Regal's rules doesn't align with your team's preferences, don't worry! Regal is not meant to be the law,
+and some rules may not make sense for your project, or parts of it.
+Regal provides several different methods to ignore rules with varying precedence.
+The available methods are (ranked High to Lowest precedence):
```

Comment:
Mostly just curious about the use of capitalized High and Lowest here! Also, why is it "High" and not "Highest"?

Reply:
Ha, oops. That went from `High to Low` to `High to Lowest precedence` to `ranked High to Lowest precedence` and I just was too careless to change the case... not that it even needed to be Title Case in the first place! 😂